internal_ws/ykbh/src/lib.rs

```diff
@@ -267,7 +278,6 @@ impl SIRInterpreter {
             Terminator::Call {
                 operand: op,
                 args,
-                destination: _dest,
```

Comment:
We can't just remove this line, so not sure what to do here.

Reply:
What's the question? We don't want a variable name with an `_`?

We can replace the line with `..,`, but I don't think it add's much value.

Reply:
I think we can just ignore this one. Just mark the remaining open discussions as resolved and I think we can close this.

Reply:
Or even just `destination: _`.